modules/assimp/assimp.cpp

```diff
@@ -0,0 +1,2181 @@
+/*************************************************************************/
+/*  assimp.cpp                                                           */
```

Comment:
That file should still be renamed to `editor_scene_importer_assimp.cpp` (same for the header), and the copyright comment edited accordingly (make sure that the trailing `*/` is still aligned afterwards).

Reply:
Renamed.